src/lib.rs

```diff
@@ -9,5 +9,7 @@ pub trait Label {
     fn lub(self, rhs: Self) -> Self;
     fn glb(self, rhs: Self) -> Self;
     fn can_flow_to(&self, rhs: &Self) -> bool;
+    #[cfg(feature = "dclabel")]
```

Comment:
This doesn't make sense to both be a trait function _and_ be behind a label-format-specific feature flag. Either this should just be a method on `DCLabel`, or it should be a method for _all_ `Label` instances (in which case `dclabel::Component` should be probably an associated type of the `Label` trait), or there should be a separate trait for labels that have a related privilege.